modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

```diff
@@ -224,6 +224,33 @@ For more information, see xref:installation-and-upgrade:container-management/per
 ====
 
 
+
+==== Troubleshhoting Full Disk Event
+
+In case a dedicated disk that was mounted as a persistent storage medium runs out of storage space, proceed as follows to solve the problem.
+
+.Procedure: Resize Storage Medium
+
+. Increase the disk (for example, [path]``/dev/vdb``) size.
+  For example, if the disk is a QCOW2 image, use the command [command]``qemu-img resize``.
+
+. Ensure the partition uses all available space:
+* open a shell in the container using [command]``mgrctl term``.
+* [command]``sudo parted /dev/vdb``
+* [literal]``(parted)`` [command]``print``
+* [literal]``(parted)`` [command]``resizepart NUMBER 100%`` where [literal]``NUMBER`` is the partition number shown by [command]``print`` command (for example, [literal]``1`` if [literal]``/dev/vdb1``)
+* [literal]``(parted)`` [command]``quit``
+. Resize the filesystem. For example, for an XFS filesystem run in a shell inside the container:
++
+
+----
+sudo xfs_growfs /dev/vdb1
+----
+
+Completing this procedure, an XFS filesystem on [literal]``/dev/vdb1`` should now be using all space available on the disk.
```

Comment:
```suggestion
After completing this procedure, a XFS filesystem on [literal]``/dev/vdb1`` should be using all space available on the disk.
```

Reply:
I'm not sure, but judging from https://en.wikipedia.org/wiki/English_articles#Distinction_between_a_and_an, I'd like to keep "an" because a vowel sound follows